includes/Classifai/Plugin.php

```diff
@@ -144,6 +145,17 @@ public function init_admin_helpers() {
 		}
 	}
 
+	/**
+	 * Adds information to the privacy policy.
+	 */
+	public function add_privacy_policy_content() {
+		$content  = '<p class="privacy-policy-tutorial">' . esc_html__( 'ClassifAI provides integrations with various AI providers. We recommend that you are transparent with your users that these AI tools are being used.', 'classifai' ) . '</p>';
```

Comment:
Perhaps `ClassifAI integrates with various AI service providers. We recommend that you are transparent with your users that these AI integrations are in use.`